testing.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM fedora
```

Comment:
so should we use this dockerfile or the one in the other repo? this one is more minimal...

I vote that we will use this one, just put this in [here](https://github.com/Bash-it/bash-it-docker/pull/3/files)

Reply:
The reason I have 2 is this one is based on fedora and is a little heavier, and the other is an alpine with only the requirements.

But I am glad to move to this as it's my go-to container as of late